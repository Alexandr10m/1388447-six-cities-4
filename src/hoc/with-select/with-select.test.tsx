import * as React from "react";
import * as renderer from "react-test-renderer";
import withSelect from "./with-select.js";


const MockComponent = () => <form/>;

const MockComponentWrapped = withSelect(MockComponent);

describe(`Snapshot of withSelect`, () => {
  it(`withSelect should render correctly`, () => {
    const tree = renderer
      .create(<MockComponentWrapped/>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
