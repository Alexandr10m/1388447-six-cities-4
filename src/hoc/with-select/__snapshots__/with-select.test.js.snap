// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of withSelect withSelect should render correctly 1`] = `<form />`;
